services/api-server/tests/unit/conftest.py

```diff
@@ -134,7 +134,7 @@ def auth(mocker, app: FastAPI, faker: Faker) -> HTTPBasicAuth:
         return_value=UserAndProductTuple(user_id=faker.pyint(), product_name="osparc"),
     )
     mocker.patch(
-        "simcore_service_api_server.db.repositories.users.UsersRepository.get_email_from_user_id",
+        "simcore_service_api_server.db.repositories.users.UsersRepository.get_active_user_email",
```

Comment:
why the word "active"?

Reply:
@bisgaard-itis because the user must be active. This is necessary, specially because now we can delete accounts or banned them

Reply:
Aha ok, makes sense. Thanks for the clarification